src/plugins/index_pattern_management/public/components/create_index_pattern_wizard/components/step_index_pattern/step_index_pattern.tsx

```diff
@@ -66,6 +63,10 @@ interface StepIndexPatternState {
 }
 
 export class StepIndexPattern extends Component<StepIndexPatternProps, StepIndexPatternState> {
+  static contextType = contextType;
```

Comment:
Is this needed? If so how is it used?

Reply:
I found the following explanation: 
> The most common way to access Context from a class component is via the static contextType. If you need the value from Context outside of render, or in a lifecycle method, you'll use it this way.

Reply:
ah, this is the context and react makes it available via `this.context`